mcstatus/bedrock_status.py

```diff
@@ -96,6 +97,6 @@ def __init__(
         self.latency = latency
         self.players_online = players_online
         self.players_max = players_max
-        self.motd = motd
+        self.motd = Motd.parse(motd)
```

Comment:
Would be nice if someone propose deprecation variant

Reply:
Same as my [previous reply](https://github.com/py-mine/mcstatus/pull/335#discussion_r906783870)